x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/source_data.tsx

```diff
@@ -59,32 +61,44 @@ import {
 
 import { FeatureIds } from '../../types';
 
+import { SOURCE_NAMES, SOURCE_OBJ_TYPES, GITHUB_LINK_TITLE } from '../../constants';
+
 const connectStepDescription = {
-  attachments:
-    'Content found within Attachments (PDFs, Microsoft Office Files, and other popular textual file formats) will be automatically indexed and searchable.',
-  files:
-    'Content found within PDFs, Microsoft Office Files, and other popular textual file formats will be automatically indexed and searchable.',
+  attachments: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.connectStepDescription.attachments',
+    {
+      defaultMessage:
+        'Content found within Attachments (PDFs, Microsoft Office Files, and other popular textual file formats) will be automatically indexed and searchable.',
+    }
+  ),
+  files: i18n.translate(
+    'xpack.enterpriseSearch.workplaceSearch.sources.connectStepDescription.files',
+    {
+      defaultMessage:
+        'Content found within PDFs, Microsoft Office Files, and other popular textual file formats will be automatically indexed and searchable.',
+    }
+  ),
   empty: '',
 };
 
 const sourceNames = {
-  confluence: 'Confluence',
-  confluenceServer: 'Confluence (Server)',
-  dropbox: 'Dropbox',
-  github: 'GitHub',
-  githubEnterprise: 'GitHub Enterprise Server',
-  gmail: 'Gmail',
-  googleDrive: 'Google Drive',
-  jira: 'Jira',
-  jiraServer: 'Jira (Server)',
-  oneDrive: 'OneDrive',
-  salesforce: 'Salesforce',
-  salesforceSandbox: 'Salesforce Sandbox',
-  serviceNow: 'ServiceNow',
-  sharePoint: 'SharePoint Online',
-  slack: 'Slack',
-  zendesk: 'Zendesk',
-  custom: 'Custom API Source',
+  confluence: SOURCE_NAMES.CONFLUENCE,
+  confluenceServer: SOURCE_NAMES.CONFLUENCE_SERVER,
+  dropbox: SOURCE_NAMES.DROPBOX,
+  github: SOURCE_NAMES.GITHUB,
+  githubEnterprise: SOURCE_NAMES.GITHUB_ENTERPRISE,
+  gmail: SOURCE_NAMES.GMAIL,
+  googleDrive: SOURCE_NAMES.GOOGLE_DRIVE,
+  jira: SOURCE_NAMES.JIRA,
+  jiraServer: SOURCE_NAMES.JIRA_SERVER,
+  oneDrive: SOURCE_NAMES.ONE_DRIVE,
+  salesforce: SOURCE_NAMES.SALESFORCE,
+  salesforceSandbox: SOURCE_NAMES.SALESFORCE_SANDBOX,
+  serviceNow: SOURCE_NAMES.SERVICE_NOW,
+  sharePoint: SOURCE_NAMES.SHARE_POINT,
+  slack: SOURCE_NAMES.SLACK,
+  zendesk: SOURCE_NAMES.ZENDESK,
+  custom: SOURCE_NAMES.CUSTOM,
 };
```

Comment:
Since this variable now doesn't store any unique data and only stores references to other variables, I think we should use those variables directly where needed.

Reply:
Great catch!